test/powersystem.jl

```diff
@@ -101,16 +101,22 @@ const PS = PowerSystem
     @testset "Computing Jacobian of residuals" begin
         F = zeros(Float64, npv + 2*npq)
         # Compute Jacobian at point V manually and use it as reference
-        J = ExaPF.residual_jacobian(V, Ybus, pv, pq)
+        J = residual_jacobian(V, Ybus, ref, pv, pq)
         J♯ = copy(J)
+        # Build the Jacobian structure
+        mappv = [i + npv for i in pv]
+        mappq = [i + npv for i in pq]
+        # Ordering for x is (θ_pv, θ_pq, v_pq)
+        statemap = vcat(mappv, mappq, pq)
```

Comment:
@frapac This is the test that is failing. It seems `pv` `pq` have a different indexing than we do internally in ExaPF, at least at the stage when AutoDiff is called.  Since that map has to be created outside of AutoDiff, I tried to use the same logic than internally in ExaPF but this fails. The resulting maps look different. Are the indices reshuffled somewhere in `PolarForm` when they come from PS?